cfme/tests/cloud/test_providers.py

```diff
@@ -453,6 +453,16 @@ def test_openstack_provider_has_api_version(appliance):
     assert view.api_version.is_displayed, "API version select is not visible"
 
 
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.10')
+def test_openstack_provider_has_dashboard(appliance, openstack_provider):
+    """Check whether dashboard view is available for Openstack provider
+    https://bugzilla.redhat.com/show_bug.cgi?id=1487142
```

Comment:
@lina-nikiforova Please don't add like this

add doc string attribute
"""Check whether dashboard view is available for Openstack provider
    bugzillas:
        * 1487142
"""
It's it will help you to track multiple BZ's on one test.

Reply:
@digitronik thanks! In this case, I decided to remove this reference to BZ, cause it's simply a new feature tracked in BZ [RFE], not a bug